### PR TITLE
Adding Icons

### PR DESCRIPTION
**Pull request by RaphaelAlvez**
Readme with link to the source

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3042/pr-test (or any other room on that server)

**Comment by ArnoldSmith86** [comment]
This doesn't use the icons yet though. IMO that should be part of the same pull request (you can add commits later).

**Comment by RaphaelAlvez** [comment]
Ready

**Comment by ArnoldSmith86** [comment]
I'm searching for 5 minutes on their site now and I can't see any license information. Am I stupid? What can we do with these images?

**Comment by ArnoldSmith86** [comment]
OK, found it: https://fontawesome.com/license/free

File: client/room.html [hunk 1]
```diff
@@ -6,19 +6,21 @@
   <title>VirtualTabletop.io</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
   <link rel="shortcut icon" href="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=="/>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
```
Comment (ArnoldSmith86) [inline client/room.html:10]: Please fix the indentation. Not a big issue but I don't want to that after each commit. :)

**Comment by ArnoldSmith86** [comment]
I think you added the font file but don't use it anywhere yet, right?

File: assets/icons-white/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Icons on this folder were found in https://fontawesome.com/
```
Comment (ArnoldSmith86) [inline assets/icons-white/README.md:1]: Please add a link to the license (https://fontawesome.com/license/free) and just say it's CC BY 4.0 directly in the file. Makes it a bit easier.

File: client/room.html (hunk 1 above)
Comment (ArnoldSmith86) [reply client/room.html:10]: Also: please actually use the font or remove the link for now.

**Review by ArnoldSmith86 — CHANGES_REQUESTED** [review]

File: assets/icons-white/README.md (hunk 2 above)
Comment (ArnoldSmith86) [inline assets/icons-white/README.md:1]: Please add a link to the license (https://fontawesome.com/license/free) and just say it's CC BY 4.0 directly in the file. Makes it a bit easier.

File: client/room.html (hunk 1 above)
Comment (ArnoldSmith86) [inline client/room.html:10]: Please fix the indentation. Not a big issue but I don't want to that after each commit. :)

Also: please actually use the font or remove the link for now.

**Review by ArnoldSmith86 — CHANGES_REQUESTED** [review]

**Review by ArnoldSmith86 — APPROVED** [review]

File: assets/icons-white/README.md (hunk 2 above)
Comment (RaphaelAlvez) [reply assets/icons-white/README.md]: Done

File: client/room.html (hunk 1 above)
Comment (RaphaelAlvez) [reply client/room.html]: Done

**Comment by RaphaelAlvez** [comment]
I consider this pull request ready. Waiting for anyone to confirm

**Pull request merged**